gravitee-management-api-service/pom.xml

```diff
@@ -66,6 +66,7 @@
 		<dependency>
 			<groupId>io.gravitee.repository</groupId>
 			<artifactId>gravitee-repository</artifactId>
+			<version>1.12.0-SNAPSHOT</version>
```

Comment:
Please define versions in the parent pom : https://github.com/gravitee-io/gravitee-management-rest-api/blob/9e00ad9630ccb7351733a7a47ec0f86c1555a039/pom.xml#L40

Reply:
done and pushed, also pushed fix for compilation error on repository-elasticsearch-*x